### ISSUE TITLE
load-mongodb.sh has a dependency on Go; Update Readme.

### ISSUE THREAD

**Issue by vinomaster** [zos-spark/interactive-insights-workbench#1]
The load-mongodb.sh script fails to complete install process of mongodb container.

Error:
Bringing up MongoDB container
Creating volume "mongodb_data" with default driver
mongodb_mongodb_1 is up-to-date
MongoDB container ID:
docker: "inspect" requires a minimum of 1 argument.
See 'docker inspect --help'.

This line fails

```
CONTAINER_ID=$(docker ps --filter ancestor=mongo --format {{.ID}})
echo MongoDB container ID: $CONTAINER_ID
```

**Comment by vinomaster** [comment]
This works:
docker ps --filter Name=mongodb_  --format {{.ID}}

Bringing up MongoDB container
Creating volume "mongodb_data" with default driver
mongodb_mongodb_1 is up-to-date
MongoDB container ID: c785a2c87591
MongoDB image: mongo:3.2
MongoDB network: mongodb_default
MongoDB container IP address: 172.19.0.2
Importing /tmp/data/client_features.csv...
2016-03-14T18:45:19.743+0000    no collection specified
2016-03-14T18:45:19.744+0000    using filename 'client_features' as collection
2016-03-14T18:45:19.746+0000    connected to: 172.19.0.2
2016-03-14T18:45:20.476+0000    imported 6001 documents

**Comment by jtyberg** [comment]
The error was due to the fact that the script was unable to retrieve the CONTAINER_ID of the running mongodb container using:

```
CONTAINER_ID=$(docker ps --filter ancestor=mongo --format {{.ID}})
```

The problem is that `docker ps` is not returning the ID because `--filter ancestor=mongo` is not working.  Using `--filter ancestor=mongo:3.2` does work, so we now include the tag, even though the [docker ps docs](https://docs.docker.com/engine/reference/commandline/ps/#filtering) say the tag is optional. 

Fixed by 1840d6a930b3aa72ab1c3fccd8d83d6db3c17bde
